internal/controller/controller.go

```diff
@@ -213,7 +213,7 @@ func (d *Controller) createUser(ctx context.Context, userAccount *AccountClaims,
 			return fmt.Errorf("failed to insert email: %w", err)
 		}
 
-		if err := d.cioService.SetEmail(ctx, acct.ID, normalEmail); err != nil {
+		if err := d.cioService.SetEmail(ctx, acct.ID, *userAccount.EthereumAddress, normalEmail); err != nil {
```

Comment:
We're creating the user, and we only have email, so`userAccount.EthereumAddress` is almost certainly nil and the handler will panic.

Reply:
I think if you could guarantee that you'll never create an account with email _first_ then we might be able to make this work. In fact, we would probably want to think about getting rid of this `id` KSUID altogether. Is this the case?

Reply:
Okay, I'm looking at [these lines](https://github.com/DIMO-Network/accounts-api/blob/cca3027ab2c71694e74f74de1b05cc27cd7a9443/internal/controller/crud_handlers.go#L51-L53). In [the logs](https://kibana.team.dimo.zone/app/r/s/rgvM9) it seems like we always start with wallet. I have no idea why the other path is there, then.